scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -94,7 +95,8 @@ class PostProcessCheckType(Enum):
     CLUSTER_STATUS = 'ClusterStatus',
     RESPONSE_VALIDATION = 'Response',
     CONSTRAINT_VALIDATION = 'Constraints',
-    SAVE_AS_VARIABLE = 'SaveAs'
+    SAVE_AS_VARIABLE = 'SaveAs',
+    WAIT_VALIDATION = 'Wait'
```

Comment:
wondering aloud: why are these `enum` items strings? Why are we not using `auto()` to just get some values? I am concerned if we ever assume these values are equal to specific strings. If we do need string formatting, maybe having a string method would be clearer intent-wise.

Reply:
There is no reasons now as far as I can tell. Fixed.